addon/templates/components/frost-sort-item.hbs

```diff
@@ -8,31 +8,25 @@
   onChange=(action 'changeProperty')
 }}
 
-{{#frost-button
-  class=(concat css '-direction')
+{{frost-button
+  class=(concat css '-direction' (if _descending ' descending'))
   hook=(concat hookPrefix '-direction')
   hookQualifiers=hookQualifiers
-  size='medium'
+  size='small'
   priority='tertiary'
+  icon='direction'
+  pack='frost-sort'
   onClick=(action 'changeDirection')
 }}
-  {{frost-icon
-    class=(if _descending 'descending')
-    hook=(concat hookPrefix '-direction-icon')
-    hookQualifiers=hookQualifiers
```

Comment:
If we were to make an update to frost-button to pass the `hookQualifiers` down into `frost-icon` we would not need to update the tests below. https://github.com/ciena-frost/ember-frost-core/blob/master/addon/templates/components/frost-button.hbs#L23-L27 Will you make this addition now so that the test does not need to be modified?

Reply:
@juwara0 addressed with a refactor